### DISCUSSION TITLE
Welcome to collaborators (PR mergers)

### DISCUSSION THREAD

**Discussion by jtr13** [jtr13/cc21fall1#2] [Announcements]
Hi All,

Welcome! As soon as we have some pull requests, I will schedule a meeting to discuss the merge process. From now on, all discussion should happen here rather than in CourseWorks.

- [x] @aye21874 (Ayush)
- [x] @clarissarjtai (Clarissa)
- [x] @ejosied (Josie)
- [x] @hwelinkim (Hyo Won)
- [x] @ivanye2509 (Xin)
- [x] @kfijan (Katharina)
- [x] @ktsht (Alex)
- [x] @mtz2110 (Maxwell)
- [x] @s10singh97 (Shashwat)
- [x] @ShiyuWang88 (Shiyu)
- [x] @Shruti-Kaushal (Shruti)
- [x] @verlocks (Zheyu)

Please react with an emoji so I know you're getting email notifications with GitHub mentions.

**Comment by Ivanye2509** [comment]
😊

**Comment by HWelinkim** [comment]
👍🏻😁

**Comment by Shruti-Kaushal** [comment]
👻☠️ 🎃
Happy Halloween!

**Reply by jtr13** [reply]
Happy Halloween! Please accept the invite to be a collaborator so you'll have write access to the repository.

**Comment by ShiyuWang88** [comment]
👍

**Comment by aye21874** [comment]
😃

**Comment by s10singh97** [comment]
👍🏻

Sent from Yahoo Mail on Android 
 
  On Sun, 31 Oct 2021 at 21:25, Joyce ***@***.***> wrote:   


Hi All,

Welcome! As soon as we have some pull requests, I will schedule a meeting to discuss the merge process. From now on, all discussion should happen here rather than in CourseWorks.

@aye21874 (Ayush), @clarissarjtai (Clarissa), @ejosied (Josie), @HWelinkim (Hyo Won), @Ivanye2509 (Xin)
@kfijan (Katharina), @ktsht (Alex), @mtz2110 (Maxwell), @s10singh97 (Shashwat), @ShiyuWang88 (Shiyu), @Shruti-Kaushal (Shruti), @verlocks (Zheyu)

Please react with an emoji so I know you're getting email notifications with GitHub mentions.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by verlocks** [comment]
😁

**Comment by jtr13** [comment]
Hi All,

If you weren't able to attend today's meeting, please watch the recording which is available in CourseWorks in the usual place for class recordings. Even though only the second half of the meeting was meant for all the PR mergers, I actually think the first half will be helpful for everyone, so in short I encourage to watch the whole session. For example, the process of merging "waffle_cheatsheet.Rmd" into the project which I demo during the first half of the meeting is relevant to everyone.

Let me know if you have any questions.

Joyce
